cmd/tsbs_generate_data/main.go

```diff
@@ -173,7 +173,7 @@ func init() {
 
 	flag.StringVar(&format, "format", "", fmt.Sprintf("Format to emit. (choices: %s)", strings.Join(formatChoices, ", ")))
 
-	flag.StringVar(&useCase, "use-case", "", "Use case to model. (choices: devops, cpu-only)")
+	flag.StringVar(&useCase, "use-case", "", "Use case to model. (choices: devops, cpu-only, cpu-single)")
```

Comment:
I'd recommend adding a "semi-constant" slice of the `useCase*` constants and do the same thing as the `--format` flag above.